test/operator.test.js

```diff
@@ -305,7 +305,41 @@ contract('SelfToken', function (accounts) {
   })
```

Comment:
The description should be "should not be able to add non contract official operator"